pkg/cmdlets/errors/errors.go

```diff
@@ -103,6 +103,16 @@ func ParseUserFacing(err error) (int, error) {
 		return code, nil
 	}
 
+	// If there is a user facing error in the error stack we want to ensure
+	// that is it forwarded to the user.
+	var userFacingError errs.UserFacingError
+	if errors.As(err, &userFacingError) {
+		logging.Debug("Returning user facing error, error stack: \n%s", errs.JoinMessage(err))
+		return code, &OutputError{userFacingError}
```

Comment:
OutputError doesn't know to look for UserError().

Reply:
Ahh okay, now I understand why I was seeing the user-facing error output as `OutputError` will eventually call the `Error()` method, but we want it to call `UserError()`. Thanks for catching that!